### PR TITLE
SEO-171719-Links-With-Redirects-UWP-Charts

### PR DESCRIPTION
**Pull request by berylouma**
Hello,

I have worked on the links with redirects and changing image names and alt text.

Regards,
Beryl.

### CODE REVIEW

File: uwp/Charts/Area.md [hunk 1]
```diff
@@ -319,7 +319,7 @@ chart.Series.Add(columnSeries2);
 
 {% endtabs %}
 
-![SfChart with multiple panes](Area_images/Area_img4.jpeg)
+![SfChart with multiple panes in UWP](Area_images/uwp-charts-area-multiple-panes.jpeg)
```
[the remark this answers is not in the document]
Comment (berylouma) [reply uwp/Charts/Area.md]: Corrected

File: uwp/Charts/Area.md [hunk 2]
```diff
@@ -166,7 +166,7 @@ chart.AreaBorderThickness = new Thickness(3);
 
 {% endtabs %}
 
-![SfChart with customized plot area](Area_images/Area_img3.jpeg)
+![SfChart with customized plot area in UWP](area_images/uwp-charts-area-customized-plot.jpeg)
```
Comment (MallikaRK) [inline uwp/Charts/Area.md:169]: we need to give meaningful image name.
Change the image name like as below,
uwp-charts-plot-area-customization

**Review by MallikaRK — CHANGES_REQUESTED** [review]
Commented

File: uwp/Charts/Area.md (hunk 2 above)
Comment (berylouma) [reply uwp/Charts/Area.md:169]: Renamed

**Review by MallikaRK — APPROVED** [review]
Approved

**Review by Girija-Rajendran — APPROVED** [review]

**Pull request merged**